src/mesh/mechanical/plane/femMesh.cpp

```diff
@@ -19,9 +19,14 @@
 
 namespace neon::mechanical::plane
 {
-femMesh::femMesh(BasicMesh const& basic_mesh, json const& material_data, json const& simulation_data)
+femMesh::femMesh(BasicMesh const& basic_mesh,
+                 json const& material_data,
+                 json const& simulation_data,
+                 double const& generate_time_step)
```

Comment:
A reference is usually 64 bits anyway, so passing it by reference doesn't save any time / space.  It's only useful to pass by reference for objects which are larger than the inbuilt types (like double, int, float).  You can just pass by value and leave the `const`